### PR TITLE
[Task] Create appliances asset

### PR DESCRIPTION
**Pull request by Lohenrex**
## Description
Our appliances do not depend on anything else to exist, but are required for creating a recipe. These assets are created by an admin.

##Screenshots
- Creating a new appliance
Creating a new appliance

- Appliances index view
Appliances index view

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Rails.application.routes.draw do
   devise_for :users
   root "home#index"
-  resources :ingredients
-  resources :users, only: %i[index edit update destroy]
+  resources :appliances, except: %i[show]
+  resources :ingredients, except: %i[show]
+  resources :ingredient_categories, except: %i[show]
```
Comment (itsmeurbi) [inline config/routes.rb:8]: No needed route 👀

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
One small thing

File: app/views/appliances/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<div class="column">
+  <h3 class="title has-text-white-ter has-text-right">
+    Appliances
+  </h3>
+  <hr class="login-hr">
+  <p class="subtitle has-text-white-ter has-text-right">
+    We already have <%= Appliance.count %> appliances!
```
Comment (luisAzcuaga) [inline app/views/appliances/index.html.erb:7]: Take a look at this other [comment](https://github.com/Lohenrex/mogu-mogu/pull/13#discussion_r794122375) to fix this :)

**Review by luisAzcuaga — CHANGES_REQUESTED** [review]
Same comment as the other PR 👀

File: config/routes.rb (hunk 1 above)
Comment (Lohenrex) [reply config/routes.rb:8]: Which one is not needed? The users one? :o

Comment (itsmeurbi) [reply config/routes.rb:8]: Do you have a `ingredient_categories` model? 👁️ 
I don't think so
Screen Shot 2022-01-28 at 10 23 51

File: app/views/appliances/index.html.erb (hunk 2 above)
Comment (Lohenrex) [reply app/views/appliances/index.html.erb]: Already fixed in the latest commit!

**Review by itsmeurbi — APPROVED** [review]
LGTM!

**Review by luisAzcuaga — APPROVED** [review]
LGTM!

**Pull request merged**